target/linux/x86/64/config-5.15

```diff
@@ -264,11 +266,13 @@ CONFIG_INTEL_IOMMU_FLOPPY_WA=y
 # CONFIG_INTEL_IPS is not set
 # CONFIG_INTEL_MEI_HDCP is not set
 # CONFIG_INTEL_MENLOW is not set
-CONFIG_INTEL_PCH_THERMAL=y
+# CONFIG_INTEL_PCH_THERMAL is not set
```

Comment:
Why do you deactivate `CONFIG_INTEL_PCH_THERMAL`?

Reply:
I understood `CONFIG_INTEL_HFI_THERMAL=y` to supply similar functionality making this setting redundant.  The model desktop distro I used to compare does not have this option enabled.  I can toggle it back if you feel it is needed.